modules.d/01systemd-repart/module-setup.sh

```diff
@@ -34,14 +34,13 @@ depends() {
 # Install the required file(s) for the module in the initramfs.
 install() {
 
-    # Install the required file(s.
+    # Install vendor repartition configurations
+    # Install the systemd type service unit for systemd repart.
+    # Install the binary executable(s) for systemd repart
     inst_multiple -o \
-        # Install vendor repartition configurations
         $libdir/repart.d/*.conf
```

Comment:
The original missed an line break and so does this. It should be -->  $libdir/repart.d/*.conf \  <--

Reply:
fixed